test/e2e/windows/BUILD

```diff
@@ -32,7 +32,6 @@ go_library(
         "//staging/src/k8s.io/kubelet/config/v1beta1:go_default_library",
         "//test/e2e/framework:go_default_library",
         "//test/e2e/framework/kubelet:go_default_library",
-        "//test/e2e/framework/log:go_default_library",
```

Comment:
The CI job `pull-kubernetes-bazel-build` continues to be failed because of
```
compile: missing strict dependencies:
	/b/f/w/test/e2e/windows/dns.go: import of "k8s.io/kubernetes/test/e2e/framework/log"
	/b/f/w/test/e2e/windows/security_context.go: import of "k8s.io/kubernetes/test/e2e/framework/log"
```
That means these go files still use e2elog but this PR removes it from BUILD file.
So we need to take care of these files also in this PR.

Reply:
I cannot find the `dns.go` and `security_context.go` in the `test/e2e/windows` pkg

```
windows
├── BUILD
├── OWNERS
├── README.md
├── density.go
├── framework.go
├── gmsa.go
├── hybrid_network.go
├── memory_limits.go
├── networking.go
├── service.go
└── volumes.go

0 directories, 11 files
```